examples/hbase-snapshot-migration-scripts/export-snapshot.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
+# This script creates and exports hbase snapshots to another cluster
+# Run this script on HBase 1x cluster
+# Usage: ./export-snapshot.sh <path/to/root/dir>
```

Comment:
Is this the root directory of the HBase 2x cluster?

Reply:
this should be what `hbase.rootdir` is set to which is usually directory in S3. I'll update the comment to specify that